### PR TITLE
Completed Exercises: Polymorphism

### PR DESCRIPTION
**Pull request by Pukkah**

### CODE REVIEW

File: src/main/java/io/codelex/polymorphism/practice/exercise1/Audi.java [hunk 1]
```diff
@@ -1,7 +1,7 @@
 package io.codelex.polymorphism.practice.exercise1;
 
-public class Audi {
-    private Integer currentSpeed = 0;
+public class Audi implements Car {
+    private int currentSpeed = 0;
 
     public void speedUp() {
```
Comment (NilsKula) [inline src/main/java/io/codelex/polymorphism/practice/exercise1/Audi.java:6]: izmanto @override pie inherited metodēm

File: src/main/java/io/codelex/polymorphism/practice/exercise6/AnimalsMain.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
+package io.codelex.polymorphism.practice.exercise6;
+
+import java.lang.reflect.Constructor;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Scanner;
+
+public class AnimalsMain {
+    private static final Map<String, Constructor> animalConstructors = Map.of(
```
Comment (NilsKula) [inline src/main/java/io/codelex/polymorphism/practice/exercise6/AnimalsMain.java:10]: caps naming

**Review by NilsKula — APPROVED** [review]

File: src/main/java/io/codelex/polymorphism/practice/exercise6/AnimalsMain.java (hunk 2 above)
Comment (Pukkah) [reply src/main/java/io/codelex/polymorphism/practice/exercise6/AnimalsMain.java:10]: Paldies, biju nedaudz apmulsis vai **private static** arī ir ALL_CAPS.

Comment (NilsKula) [reply src/main/java/io/codelex/polymorphism/practice/exercise6/AnimalsMain.java:10]: any kind of static final 100% 😄

**Pull request merged**